### DISCUSSION TITLE
Issue with Dropdown Component in Shadcn: Accessibility Problem

### DISCUSSION THREAD

**Discussion by RushikeshChamle** [shadcn-ui/ui#2915] [Q&A] [answered]
Hello Shadcn Community,

I'm encountering an issue with the dropdown component in Shadcn. When I set the dropdown to stay open using <DropdownMenu open={true}>, I'm unable to access other page components while it remains open.

Could you please advise on the problem and suggest a resolution?

Thank you!

**Comment by pablotanner** [comment]
It's because a Dropdown behaves like a modal, disabling any pointer-events on other elements while its open. If you want to disable this behaviour, you can add the 'modal={false}' property in DropdownMenu.

```javascript
<DropdownMenu modal={false}>
                <DropdownMenuTrigger asChild className="cursor-pointer">
                    <MoreHorizontal className="h-5 w-5 ml-3"/>
                </DropdownMenuTrigger>
                <DropdownMenuContent>
                ... rest of code
```

**Reply by RushikeshChamle** [reply, marked as answer]
This is working as expected now 
Thanks, man 

✨✨
